### PR TITLE
mnist_test.py PASSED

### PR DESCRIPTION
**Pull request by akshaychawla**
1. Scale data to data /= max(data)
2. MSE is not a good loss, starts out flat, then starts learning
properly.
3. Use an aggressive learning rate.
4. Train for 500 epochs.

### CODE REVIEW

**Review by priyamtejaswin — APPROVED** [review]
works!
